packages/transition-frontend/src/components/forms/accessibilityComparison/AccessibilityComparisonForm.tsx

```diff
@@ -504,45 +571,122 @@ class AccessibilityComparisonForm extends ChangeEventsForm<AccessibilityComparis
+                                                choices={scenarios}
+                                                t={this.props.t}
+                                                onValueChange={(e) => {
+                                                    this.onValueChange('alternateScenario2Id', {
+                                                        value: e.target.value
+                                                    });
+                                                    alternateRouting.attributes.scenarioId = e.target.value;
+                                                }}
+                                            />
+                                        </InputWrapper>
+                                    </div>
+                                    <AccessibilityMapCoordinatesComponent
+                                        id={'formFieldTransitAccessibilityMapComparisonLocation'}
+                                        locationGeojson={this.state.object.attributes.locationGeojson}
+                                        onUpdateCoordinates={this.onUpdateCoordinatesForBothEngines}
                                     />
-                                </InputWrapper>
-                                <InputWrapper
-                                    label={this.props.t('transit:transitComparison:ScenarioN', { scenarioNumber: '2' })}
-                                >
-                                    <InputSelect
-                                        id={`formFieldTransitAccessibilityMapScenario2${routingId}`}
-                                        value={alternateRouting.attributes.scenarioId}
-                                        choices={scenarios}
-                                        t={this.props.t}
-                                        onValueChange={(e) => {
-                                            this.onValueChange('alternateScenario2Id', { value: e.target.value });
-                                            alternateRouting.attributes.scenarioId = e.target.value;
-                                        }}
+                                    <InputWrapper label={this.props.t('transit:transitRouting:PlaceName')}>
+                                        <InputString
+                                            id={`formFieldTransitAccessibilityMapPlaceName${routingId}`}
+                                            value={routing.attributes.placeName}
+                                            onValueUpdated={(value) =>
+                                                this.updateBothRoutingEngines('placeName', value, alternateRouting)
+                                            }
+                                        />
+                                    </InputWrapper>
+                                </React.Fragment>
+                            )}
+                            {mode === 'locations' && (
```

Comment:
Il faudra réfléchir à des sous-composants ici. Pour l'instant, il n'y a que 2 modes, mais au prochain, on passe à autre chose! Tu peux peut-être ajouter un TODO à la prochain itération à cette PR